### ISSUE TITLE
Non-User accounts

### ISSUE THREAD

**Issue by Recoil1980** [kevinpapst/kimai2#1936]
**Is your feature request related to a problem? Please describe.**
Active users with no hours show on reports, such as the owner and system-admin account.  Setting an account to inactive disables access, and there is no option to have certain accounts separate which to not generate on reports.

**Describe the solution you'd like**
Option for certain roles or accounts that would not have timesheets or calculate on any hours, however, still retain higher system admin or administrator access.

**Describe alternatives you've considered**
Revoking the owners permissions...

**Additional context**
None.

**Comment by kevinpapst** [comment]
Good feature request, I like that one! Flagging a user as "system account" could prevent it from showing up in:
- Reports
- Team assignments?
- Where else?

**Comment by Recoil1980** [comment]
Maybe User selection when creating a timesheet or expense?

A simple solution (I think) would be to add an additional boolean for the users table `is_system_account` to check against when generating the reports and fields, instead of a completely separate user-type, but looking back that seems not-so-simple.  Perhaps a set `system_account` that cannot be deleted is another thought.

I'm still working through trying to learn the ropes, and pouring over the code to gather thoughts on how some of my ideas may work fluidly with the current system.

**Comment by kevinpapst** [comment]
Yes to "boolean flag" and yes to "cannot be deleted" from my side.

**Comment by Recoil1980** [comment]
I will see if I can get a dev server setup this week to help with the code.

**Comment by mihailescu2m** [comment]
👍

**Comment by crm-developer** [comment]
I take

**Comment by crm-developer** [comment]
in practice This refers to the permission.
if we make it a flag it will complicate usability.

you need Something like this?:
![Снимок экрана от 2021-03-02 16-53-52](https://user-images.githubusercontent.com/54503325/109658705-f106a000-7b77-11eb-8375-9ea48379d675.png)

**Comment by kevinpapst** [comment]
Nope, [as written above](https://github.com/kevinpapst/kimai2/issues/1936#issuecomment-683299128): a boolean field on the user entity.

**Comment by crm-developer** [comment]
ok
